services/src/handlers/plots.rs

```diff
@@ -152,13 +152,15 @@ async fn get_plot_handler<C: Context>(
     };
 
     let query_rect = if request_spatial_ref == workflow_spatial_ref {
-        query_rect
+        Some(query_rect)
     } else {
         reproject_query(query_rect, workflow_spatial_ref, request_spatial_ref)
             .map_err(From::from)
             .context(error::Operator)?
     };
 
+    let query_rect = query_rect.expect("there is no way to return an empty plot, yet."); // TODO: discuss if the graph should be reprojected
```

Comment:
we must not `panic!` here

Could you return an error?

Reply:
[3d0fc53](https://github.com/geo-engine/geoengine/pull/656/commits/3d0fc536af26fecfa53a0215273088685f41b2e4)